src/posthog-featureflags.js

```diff
@@ -51,14 +51,17 @@ export class PostHogFeatureFlags {
      *     if(posthog.isFeatureEnabled('beta-feature')) { // do something }
      *
      * @param {Object|String} prop Key of the feature flag.
+     * @param {Bool} prop Whether to send a .
      */
-    isFeatureEnabled(key) {
+    isFeatureEnabled(key, send_event = true) {
```

Comment:
Would it make sense to make this into ```isFeatureEnabled(key, options = {})```?

This way the calling code will make a bit more sense: `.isFeatureEnabled('feature', false)` is rather opaque, `.isFeatureEnabled('feature', { send_event: false })` gives an idea of what's going on

Reply:
Yep good point, done